src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class ApplicationInsightsExtensions
     {
         public static IApplicationBuilder UseApplicationInsightsRequestTelemetry(this IApplicationBuilder app)
```

Comment:
For the second question - we discussed whether to use ApplicationInsights term in the name or not even when it was in our own namespace. The main idea of using it was to increase code readability - you need to understand that this is Application Insights telemetry that this middleware is collecting. Same way as Facebook authentication has Facebook in the name even if it will be placed in Facebook namespace.